webClient/src/app/shared/editor-control/editor-control.service.ts

```diff
@@ -208,8 +215,25 @@ export class EditorControlService implements ZLUX.IEditor, ZLUX.IEditorMultiBuff
     return this._openFileList;
   }
 
+  private saveFilePathSessionRestore(fileContext: ProjectContext){
```

Comment:
maybe updateSavedTabsStorage is a little better of a function name for others to understand the purpose of this function

Reply:
Done